sno/working_copy.py

```diff
@@ -141,6 +141,12 @@ def session(self, bulk=0):
                 del self._db
                 L.debug(f"session(bulk={bulk}): new/done")
 
+    def is_dirty(self):
```

Comment:
I wrote this too, but it turns out you don't need it if you just reset(force=force) somewhere early on in your command - it will either fail, or continue with a clean working copy, which is what you want.